src/app/components/feed/SaveFeed.js

```diff
@@ -473,10 +570,49 @@ const SaveFeed = (props) => {
                 description="On feed edit page, show the last time the feed was changed. The placeholder 'timeAgo' is something like '10 minutes ago'."
               />
             </div>
-          </div> : null
+          </div>
+          : null
         }
       </div>
 
+      {/* "Delete" dialog */}
+      <ConfirmProceedDialog
+        open={showDeleteDialog}
+        title={
+          <FormattedMessage
+            id="projectsComponent.deleteSharedFeed"
+            defaultMessage="Delete Shared Feed?"
+            description="'Delete' here is an infinitive verb"
+          />
+        }
+        body={
+          <div className={styles.saveFeedDialogDivider}>
+            <Typography variant="body1" component="p" paragraph>
+              <FormattedHTMLMessage
```

Comment:
@danielevalverde if you add `tagName="p"` to this, you can remove the `<Typography...` wrapper and also remove the reference to mui typography. We have global css rules that will format it correctly